src/Compiler/Driver/ReuseTcResults/TcResultsImport.fs

```diff
@@ -0,0 +1,133 @@
+﻿module internal FSharp.Compiler.ReuseTcResults.TcResultsImport
+
+open FSharp.Compiler.CompilerConfig
+open FSharp.Compiler.IO
+open FSharp.Compiler.CompilerImports
+open FSharp.Compiler.TypedTreePickle
+open FSharp.Compiler.ReuseTcResults.TcResultsPickle
+
+let GetSharedData (file, ilScopeRef, ilModule, byteReaderA, byteReaderB) =
+
+    let memA = byteReaderA ()
```

Comment:
This is really just copypaste from existing imports... we likely don't need to byte readers and all that stuff can be simplified.